assets/blocks/course-progress-block/course-progress-edit.js

```diff
@@ -76,18 +76,17 @@ export const CourseProgressEdit = ( props ) => {
 			borderRadius,
 		},
 	};
-
+	const progressBarLabel = _n( 'lesson', 'lessons', 2, 'sensei-lms' );
 	return (
 		<>
 			<ProgressBar
-				lessonsCount={ totalLessonsCount }
+				totalCount={ totalLessonsCount }
 				completedCount={ completedLessonsCount }
 				wrapperAttributes={ wrapperAttributes }
 				barWrapperAttributes={ barWrapperAttributes }
 				barAttributes={ barAttributes }
 				countersClassName="wp-block-sensei-lms-progress-heading"
-				lessonsCountClassName="wp-block-sensei-lms-progress-heading__lessons"
-				completedCountClassName="wp-block-sensei-lms-progress-heading__completed"
+				progressBarLabel={ progressBarLabel }
 			/>
```

Comment:
I removed this one since I've added 'wp-block-sensei-lms-progress-heading__completed' class name to the component